.github/DEVELOPMENT.md

```diff
@@ -1,5 +1,13 @@
 # Development
 
+_Contributors have two options for running and developing this project: locally on your own computer, or using GitHub Codespaces.
+GitHub Codespaces provides a cloud-hosted development environment, which means you can work on the project without needing to set up anything on your own machine.
+The instructions below will guide you through both options, allowing you to choose the method that best suits your needs._
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/OpenContributionsProject/opencontributionssite?devcontainer_path=/.devcontainer/basics/devcontainer.json)
```

Comment:
[Docs] This is an _excellent_ description of Codespaces, but I worry that placing so much explanatory marketing text so early in the document might come across as us advertising for GitHub. I'm a little paranoid about walking that balance between explaining vs. marketing, especially since you and I are known "Microsofties"-ish (past employee + current MVP; current employee + dev advocate).

What do you think about making a quick summary here & moving the Codespaces explanation & button to the _GitHub Codespaces_ section?

```suggestion
We provide two preset ways to work on this repository that you can choose from:

* [Local Development](#local-development): Setting up the packages on your local machine
* [GitHub Codespaces](#github-codespaces): Using a cloud-hosted development environment
```

Reply:
I like this idea @JoshuaKGoldberg! :) It will make things less redundant! :)

Reply:
sounds good! My brain is too GitHub allll the time!